codespell_lib/data/dictionary.txt

```diff
@@ -6187,6 +6187,8 @@ coomand->command
 coommand->command
 coonstantly->constantly
 coonstructed->constructed
+coopearte->cooperate
+coopeartes->cooperates
```

Comment:
Also suggest the hyphenated version, for non Americans.

Reply:
Personally I was meaning suggest either alternative, i.e.
`coopearte->cooperate, co-operate,`

But I guess others may feel that's too much language translation.